src/blog/2024/11/bridging-modbus-to-mqtt.md

```diff
@@ -0,0 +1,248 @@
+6. Once saved, copy the newly added client's Client ID from the list and save it for later use.
+
+**Step 3.2: Configure MQTT Nodes in Node-RED**
+
+1. In the Node-RED editor, drag the **mqtt out** node onto the canvas.
+2. Double-click the node, then click the **+** icon next to the **Server** field to configure your broker details:
+   - For FlowFuse MQTT, use `broker.flowfuse.cloud` as the server address.
+   - Enter the **Client ID**, **username**, and **password** that you generated earlier.
+
+![Image mqtt broker node configuration](./images/mqtt-broker-config.png){data-zoomable}
+_Image mqtt broker node configuration_
+
+3. Click **Add** to save the configuration.
+4. Select the appropriate **QoS** (Quality of Service) level based on your needs.
+5. Enter the Topic for the MQTT message, such as `plant2/Area4/Cell2/DeviceA/temperature`. This naming convention organizes data by plant, area, cell, and device, making it easier to manage, filter, and scale your system. It also simplifies data segmentation and provides straightforward access when monitoring specific devices or cells.
+
+![Image showing the MQTT Out node configuration ](./images/mqtt-client-config-temperature.png){data-zoomable}
+_Image showing the MQTT Out node configuration_
+
+*Note: For the configuration of MQTT nodes and Modbus, environment variables are used to prevent accidental exposure when sharing. For more information, refer to [Using Environment Variables with Node-RED](/blog/2023/01/environment-variables-in-node-red/).*
+
+6. Click **Done** to save the node.
+7. Connect the output of the Change node, which is transforming the metrics you are sending via MQTT, to the mqtt out node.
+
+Repeat these steps for each metric (e.g., pressure, vibration, humidity), giving each one its own **mqtt out** node with a unique topic.
+
+For more information on using MQTT with Node-RED, please read [Using MQTT with Node-RED](/blog/2024/06/how-to-use-mqtt-in-node-red/).
+
+Once you’ve configured the MQTT nodes for all your metrics and deployed the flow, check the status at the bottom of each MQTT node. If it shows "connected," your Node-RED flow is successfully connected to the broker and publishes data to the MQTT broker. From here, you can integrate the data with cloud-based analytics platforms. Build IoT dashboards with [FlowFuse Dashboard](https://dashboard.flowfuse.com/), or other systems to enable real-time monitoring, predictive maintenance, and automated decision-making. This setup effectively bridges the gap between legacy Modbus devices and modern IoT infrastructure, empowering smarter, more efficient industrial operations.
+
+To monitor the data being sent to the Broker, you can use MQTT client monitoring software, such as MQTT Explorer, or similar tools.
+
+![Image showing the MQTT Explorer interface monitoring an MQTT broker](./images/mqtt-client-explorer.png){data-zoomable}
+_Image showing the MQTT Explorer interface monitoring an MQTT broker_
+
+## Final Thought
+
```

Comment:
There's no mention of FlowFuse at all in this conclusion. I would suggest editing this part accordingly.